data/DataType/boolean_toml.yaml

```diff
@@ -0,0 +1,4 @@
+keyword: boolean
```

Comment:
there are a lot of duplicated ones, so i'll clean them all.

Reply:
this PR should only contain things directly relatd to TOML.

And I forgot TOML doesnt have keywords, so it cant use the datatypes which declarea keyword

Reply:
hmm... actually keyword of type is what identifies the type. So I think it is ok to declare keyword of type.
However, what I meant is that since the doc calls it as [Boolean](https://github.com/toml-lang/toml#boolean), the keyword should be `boolean` but there is no datatype that has same keyword and same attribute.

Reply:
no, a keyword here refers to a keyword in the language.  TOML doesnt have one.  These need to be removed.

We would need to enhance the schema to define a datatype type, or datatype name, to describe what the language calls it.